docs/reference/server-configuration.md

```diff
@@ -10,10 +10,16 @@ keywords: ksqldb, configure, server, setup, install
 
 These configuration parameters control the general behavior of ksqlDB server.
 Many parameters can only be set once for the entire server, and must be
-specified using the `ksql-server.properties` file. Some parameters, however,
-can be set on a per-persistent query basis using `SET`. This is indicated in each parameter
-section below.
-Retrieve the current list of configuration settings by using the [SHOW PROPERTIES](/developer-guide/ksqldb-reference/show-properties/) command.
+specified using the `ksql-server.properties` file.
+
+You can assign the value of some parameters on a per-persistent query basis
+by using the `SET` statement. This is indicated in the following parameter
+sections with the **Per query** block. For ksqlDB in {{ site.ccloud }}, some 
+parameters can be set only by using the ALTER SYSTEM statement, and this is
+indicated in the corresponding **Per query** block.
+
+Retrieve the current list of configuration settings by using the
+[SHOW PROPERTIES](/developer-guide/ksqldb-reference/show-properties/) command.
```

Comment:
@rodesai why do we have both `SET PROPERTIES` and `LIST PROPERTIES`? Is one of them currently deprecated or going to be phased out?